obs-ndi.spec

```diff
@@ -0,0 +1,107 @@
+#*********************************************************************************************************
+#*   __     __               __     ______                __   __                      _______ _______   *
+#*  |  |--.|  |.---.-..----.|  |--.|   __ \.---.-..-----.|  |_|  |--..-----..----.    |       |     __|  *
+#*  |  _  ||  ||  _  ||  __||    < |    __/|  _  ||     ||   _|     ||  -__||   _|    |   -   |__     |  *
+#*  |_____||__||___._||____||__|__||___|   |___._||__|__||____|__|__||_____||__|      |_______|_______|  *
+#* http://www.blackpantheros.eu | http://www.blackpanther.hu - kbarcza[]blackpanther.hu * Charles Barcza *
+#*************************************************************************************(c)2002-2019********
+# 	Soec written by Charles K Barcza for blackPanther OS - www.blackpanther.hu
+
+%define genlibndi 1
+
+Name: 	 obs-ndi
+Version: 4.6.0
+Release: %mkrel 2
+Source0: %name-%version.tar.gz
+Source1: obs-studio-23.1.0-rc1.tar.gz
```

Comment:
For Fedora this is provided by:

https://download1.rpmfusion.org/free/fedora/releases/29/Everything/x86_64/os/repoview/obs-studio-devel.html